scooby/report.py

```diff
@@ -277,14 +275,12 @@ def cols(html, version, name, ncol, i):
 
         # ########## Platform/OS details ############
         html += "  <tr>\n"
-        html, i = cols(html, self.system, 'OS', self.ncol, 0)
-        html, i = cols(html, self.cpu_count, 'CPU(s)', self.ncol, i)
-        html, i = cols(html, self.machine, 'Machine', self.ncol, i)
-        html, i = cols(html, self.architecture, 'Architecture', self.ncol, i)
-        if TOTAL_RAM:
```

Comment:
Need to keep `TOTAL_RAM`.  How about

```python
keys = ['OS', 'CPU(s)', 'Machine', 'Architecture', 'Environment']
if TOTAL_RAM:
    keys.append('RAM')
for key in keys:
    # ...
```

Reply:
But 'RAM' is only in the `repr_dict()` if `TOTAL_RAM`

Reply:
(this check is now just further down in `to_dict` - or do I miss something?)

Reply:
This is why it has the line `if key in repr_dict`

Reply:
Sorry, missed that.

Reply:
No problem at all